Sources/LatiFlex/Views/Network/NetworkView/LatiFlexNetworkViewController.swift

```diff
@@ -90,7 +90,12 @@ extension LatiFlexNetworkViewController: UICollectionViewDelegateFlowLayout {
 }
 
 extension LatiFlexNetworkViewController: UISearchBarDelegate {
+
+    func searchBarSearchButtonClicked(_ searchBar: UISearchBar) {
+        presenter.textDidChange(searchtext: searchBar.text ?? "")
```

Comment:
`presenter. searchBarSearchButtonClicked` there should be a different func for this